Rocket.Chat/Controllers/Auth/ConnectServerViewController.swift

```diff
@@ -125,6 +125,23 @@ final class ConnectServerViewController: BaseViewController {
         guard let socketURL = url.socketURL() else { return alertInvalidURL() }
         guard let validateURL = url.validateURL() else { return alertInvalidURL() }
 
+        if let servers = DatabaseManager.servers {
+            let sameServerIndex = servers.index(where: {
+                if let stringServerUrl = $0[ServerPersistKeys.serverURL],
+                    let serverUrl = URL(string: stringServerUrl) {
+
+                    return serverUrl == socketURL
+                } else {
+                    return false
+                }
+            })
+
+            if let sameServerIndex = sameServerIndex {
+                MainChatViewController.shared?.changeSelectedServer(index: sameServerIndex)
```

Comment:
What if `MainChatViewController.shared` doesn't exists here?

Reply:
Sorry, but I don't know... What do you prefer that I do?